python/lsst/ts/IntegrationTests/configs/image_taking_configs.py

```diff
@@ -38,3 +38,105 @@
     explicit_start=True,
     canonical=True,
 )
+
+# auxtel_latiss_calibrations
+registry["latiss_calibrations_flat"] = yaml.safe_dump(
+    {
+        "n_bias": 10,
+        "n_dark": 10,
+        "exp_times_dark": 10,
+        "n_flat": 10,
+        "exp_times_flat": 2,
+        "filter": "SDSSg",
+        "calib_collection": "LATISS/calib/u/mareuter/daily.replace_me.calib_type",
```

Comment:
Should generalize from `mareuter` to `integrationtester`. Same comment applies to all the entries below too.